hack/overrides/jaeger.yml

```diff
@@ -0,0 +1,30 @@
+# jaeger.yml - a docker-compose override that adds 'jaeger to the stack.
+#
+# once running, head to `localhost:25000` to get access to the Jaeger UI.
```

Comment:
wondering why `25000` instead of jaegar's default port?

Reply:
25000 was a number that I could remember lol no big reason hahah `16686` in the middle of all of those other ports seem kinda hard:

```
0.0.0.0:25000->16686/tc
0.0.0.0:14268->14268/tcp	
0.0.0.0:5775->5775/udp
0.0.0.0:5778->5778/tcp
0.0.0.0:6831-6832->6831-6832/udp
0.0.0.0:9411->9411/tcp
14250/tcp
```